packages/cards/lib/card.json

```diff
@@ -275,6 +275,11 @@
       "name": "Athena IDProtect - Cryptographic Java Card",
       "driver": "29a2bd60c4b841c686be68d2874c0c02"
     },
+    {
+      "atr": "3BDF18008131FE7D006B020C0182010101434E53103180EC",
```

Comment:
@defacto64 Please check it. Is it correct?

Reply:
I have reviewed https://github.com/PeculiarVentures/webcrypto-local/blob/p11-attributes/packages/cards/lib/card.json and it seems correct to me, as regards reception of my issues #259 and #260.